docs/reference/module-api/module-overview.md

```diff
@@ -321,27 +331,63 @@ Each of these function sections takes the module, as `self`, as an argument. Thi
 | [`extendHandlers`](#extendhandlers-self) | Extend base class server-side event handlers |
 | [`queries`](#queries-self-query) | Add database query methods |
 | [`extendQueries`](#extendqueries-self) | Extend base class database query methods |
-| [`middleware`](#middleware-self) | REPLACE-ME |
-| [`tasks`](#tasks-self) | REPLACE-ME |
+| [`middleware`](#middleware-self) | Add standard Express middleware to be called on *every* request. |
+| [`tasks`](#tasks-self) | Add task functions that can be run from the CLI. |
 
 ### The extension pattern
 
-Several of these sections use an extention pattern (the sections prefixed with "extend"). These sections are used to add functionality to matching features of the module's base class. Functions in `extendMethods` enhance identically named functions in the base class' `methods` section, `extendApiRoutes` enhances API routes from the base class' `apiRoutes`, and so on.
+Several of these sections use an extension pattern. The sections prefixed with `extend` add functionality to base class sections without the prefix. For example:
+
+- functions in `extendMethods` enhance identically named functions in the base class' `methods` section, and
+- `extendApiRoutes` enhances API routes from the base class' `apiRoutes`
 
-Each individual function included in the extension section takes a `_super` argument in addition to the same arguments as the original function. `_super` is the original function, which should be called within the new extension.
```

Comment:
If you don't need to call the original function at all, just redeclare it in `methods` to override it.

Reply:
This is addressed in the `::: warning` note at the end of the section.